Userland/Shell/Shell.cpp

```diff
@@ -1759,10 +1759,11 @@ void Shell::notify_child_event()
             }
             if (child_pid == 0) {
                 // If the child existed, but wasn't dead.
-                if (job.is_suspended() && job.shell_did_continue()) {
+                if (job.is_suspended() || job.shell_did_continue()) {
                     // The job was suspended, and we sent it a SIGCONT.
```

Comment:
Maybe change the comment to say that *someone* send it a SIGCONT? Not necessarily the shell.

Reply:
The first one ("is the job suspended?") basically means that.
We can't know if this SIGCHLD is _actually_ because of a SIGCONT, but the assumption can't really go wrong, it was either continued, which means we're right, or killed, which means the job will be deleted after this loop.

Reply:
Oh you're talking about the comment
Yeah, lemme do that

Reply:
Thanks for catching that :)